sql/handler.h

```diff
@@ -1530,9 +1530,9 @@ struct handlerton
                             const char *query, uint query_length,
                             const char *db, const char *table_name);
 
-   void (*abort_transaction)(handlerton *hton, THD *bf_thd,
-			    THD *victim_thd, my_bool signal);
-   int (*set_checkpoint)(handlerton *hton, const XID* xid);
+   void (*abort_transaction)(handlerton *hton, THD *bf_thd, THD *victim_thd,
+                             my_bool signal) __attribute((nonnull));
```

Comment:
It seems that this file is missing `#include <my_attribute.h>`, to provide a compatibility `#define` for Microsoft Windows.